components/renku_data_services/message_queue/orm.py

```diff
@@ -31,6 +32,8 @@ class EventORM(BaseORM):
     id: Mapped[int] = mapped_column(primary_key=True, default=None, init=False)
     """Unique id of the event."""
 
+    timestamp: Mapped[datetime] = mapped_column("timestamp", DateTime())
```

Comment:
Is this UTC or not? I think it is probably better to just use UTC. Then you are sure that a different timezone did not get applied on a different deployment or in a different db/image.